### ISSUE TITLE
Spyder 5.4.1 failed to start with ValueError: not enough values to unpack (expected 2, got 1)

### ISSUE THREAD

**Issue by aber69** [spyder-ide/spyder#20289]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [x] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [x] Could not reproduce inside ``jupyter qtconsole`` (if console-related) - runs without any problems
* [x] Tried basic troubleshooting (if a bug/error)
    * [x] Restarted Spyder
    * [x] Reset preferences with ``spyder --reset``
    * [x] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [x] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
I have tried to install spyder=5.4.1 in docker. In the same docker I can install and run spyder=5.4.0 or spyder=5.3.3 from conda-forge, conda main (for 5.3.3) and also from pip (into the same conda main based installation). 
It crashed on starting with error described bellow...

### What steps reproduce the problem?

1. Install spyder=5.4.1 by conda-forge or pip 
2. Try to run
3. After the splash window it crashes

### Paste Traceback/Error Below (if applicable)

  
```python-traceback

PASTE TRACEBACK HERE
  File "/home/user/miniconda/envs/39/bin/spyder", line 11, in <module>
    sys.exit(main())
  File "/home/user/miniconda/envs/39/lib/python3.9/site-packages/spyder/app/start.py", line 267, in main
    mainwindow.main(options, args)
  File "/home/user/miniconda/envs/39/lib/python3.9/site-packages/spyder/app/mainwindow.py", line 1822, in main
    mainwindow = create_window(MainWindow, app, splash, options, args)
  File "/home/user/miniconda/envs/39/lib/python3.9/site-packages/spyder/app/utils.py", line 289, in create_window
    main.setup()
  File "/home/user/miniconda/envs/39/lib/python3.9/site-packages/spyder/app/mainwindow.py", line 773, in setup
    PLUGIN_REGISTRY.register_plugin(self, PluginClass,
  File "/home/user/miniconda/envs/39/lib/python3.9/site-packages/spyder/api/plugin_registration/registry.py", line 342, in register_plugin
    instance = self._instantiate_spyder5_plugin(
  File "/home/user/miniconda/envs/39/lib/python3.9/site-packages/spyder/api/plugin_registration/registry.py", line 184, in _instantiate_spyder5_plugin
    plugin_instance = PluginClass(main_window, configuration=CONF)
  File "/home/user/miniconda/envs/39/lib/python3.9/site-packages/spyder/api/plugins/new_api.py", line 331, in __init__
    container.setup()
  File "/home/user/miniconda/envs/39/lib/python3.9/site-packages/spyder/plugins/pythonpath/container.py", line 55, in setup
    self._load_pythonpath()
  File "/home/user/miniconda/envs/39/lib/python3.9/site-packages/spyder/plugins/pythonpath/container.py", line 136, in _load_pythonpath
    system_path = get_system_pythonpath()
  File "/home/user/miniconda/envs/39/lib/python3.9/site-packages/spyder/plugins/pythonpath/utils.py", line 31, in get_system_pythonpath
    env = get_user_env()
  File "/home/user/miniconda/envs/39/lib/python3.9/site-packages/spyder/utils/environ.py", line 71, in get_user_env
    env_dict = get_user_environment_variables()
  File "/home/user/miniconda/envs/39/lib/python3.9/site-packages/spyder/utils/environ.py", line 63, in get_user_environment_variables
    k, v = kv.split('=', 1)
ValueError: not enough values to unpack (expected 2, got 1)

```

## Versions

* Spyder version: 5.4.1
* Python version: 3.9
* Qt version: 5.15.9 (h06a4308_0 - conda main)
* PyQt version: 5.15.7           py39h6a678d5_1
* Operating System name/version: ubuntu 22 or ubuntu 20 in docker

### Dependencies

```
PASTE DEPENDENCIES HERE
```
dependencies: 
  - atomicwrites >=1.2.0   1.4.1 (OK)             pyhd8ed1ab_0    conda-forge
  - chardet >=2.0.0           5.1.0 (OK)            py39hf3d152e_0    conda-forge
  - cloudpickle >=0.5.0     2.0.0 (OK)             pyhd3eb1b0_0
  - cookiecutter >=1.6.0    2.1.1 (OK)            pyh6c4a22f_0    conda-forge
  - diff-match-patch >=20181111    20200713(OK)           pyh9f0ad1d_0    conda-forge
  - intervaltree >=3.0.2         3.0.2  (OK)                    py_0    conda-forge
  - ipython >=7.31.1,<9.0.0    8.7.0  (OK)          py39h06a4308_0
  - jedi >=0.17.2,<0.19.0       0.18.1 (OK)           py39h06a4308_1
  - jellyfish >=0.7                   0.9.0            py39hb9d737c_1    conda-forge
  - jsonschema >=3.2.0
  - keyring >=17.0.0             23.4.0           py39h06a4308_0
  - nbconvert >=4.0
  - numpydoc >=0.6.0        1.5.0              pyhd8ed1ab_0    conda-forge
  - parso >=0.7.0,<0.9.0     0.8.3              pyhd3eb1b0_0
  - pexpect >=4.4.0           4.8.0              pyhd3eb1b0_3
  - pickleshare >=0.4
  - psutil >=5.3
  - pygments >=2.0
  - pylint >=2.5.0,<3.0          2.15.9             pyhd8ed1ab_0    conda-forge
  - pylint-venv >=2.1.1            2.3.0              pyhd8ed1ab_0    conda-forge
  - pyls-spyder >=0.4.0       0.4.0              pyhd8ed1ab_0    conda-forge
  - pyqt >=5.6,<5.16            5.15.7   (OK)        py39h6a678d5_1
  - pyqtwebengine <5.16     5.15.7   (OK)        py39h6a678d5_1
  - python >=3.9,<3.10.0a0   3.9.15
  - python-lsp-black >=1.2.0                 1.2.1  OK)            pyhd8ed1ab_0    conda-forge
  - python-lsp-server >=1.7.0,<1.8.0     1.7.0  (OK)              hd8ed1ab_0    conda-forge
  - python_abi 3.9.* *_cp39                   3.9     (OK)                 2_cp39    conda-forge
  - pyxdg >=0.26                      0.28               pyhd8ed1ab_0    conda-forge
  - pyzmq >=22.1.0                23.2.0           py39h6a678d5_0
  - qdarkstyle >=3.0.2,<3.1.0    3.0.3              pyhd8ed1ab_0    conda-forge
  - qstylizer >=0.2.2                0.2.2              pyhd8ed1ab_0    conda-forge
  - qtawesome >=1.2.1             1.2.2              pyhd8ed1ab_0    conda-forge
  - qtconsole >=5.4.0,<5.5.0      5.4.0              pyhd8ed1ab_0    conda-forge
  - qtpy >=2.1.0                           5.15.7           py39h6a678d5_1
  - rtree >=0.9.7              1.0.1            py39hb102c33_1    conda-forge  
  - setuptools >=49.6.0    65.5.0           py39h06a4308_0
  - sphinx >=0.6.6              5.0.2              pyh6c4a22f_0    conda-forge
  - spyder-kernels >=2.4.1,<2.5.0    2.4.1           unix_pyhd8ed1ab_0    conda-forge
  - textdistance >=4.2.0      4.5.0              pyhd8ed1ab_0    conda-forge
  - three-merge >=0.1.1      0.1.1              pyh9f0ad1d_0    conda-forge
  - watchdog >=0.10.3         2.2.0            py39hf3d152e_0    conda-forge

**Comment by aber69** [comment]
Even in more simple case: 
`import spyder.utils.environ
env_dict = spyder.utils.environ.get_user_environment_variables()
ValueError: not enough values to unpack (expected 2, got 1)
`

**Comment by ccordoba12** [comment]
Hey @aber69, thanks for reporting. This is a duplicate of issue #20097 and we'll fix it in our next version (5.4.2), to be released in January,

You said:

> Work around is not to use this variable from my side...

Yep, that's what you have to do for now if you want to use 5.4.1
